lib/honeybadger.ex

```diff
@@ -174,13 +174,25 @@ defmodule Honeybadger do
           Honeybadger.notify(exception)
       end
 
+  Send a notification directly from a string, which will be sent as a
+  `RuntimeError`:
+
+      iex> Honeybadger.notify("custom error message")
+      :ok
+
+  Send a notification as a `class` and `message`:
+
+      iex> Honeybadger.notify(%{class: "SpecialError", message: "custom message"})
+      :ok
+
   If desired additional metadata can be provided as well:
 
-      Honeybadger.notify(%MyException{}, %{culprit_id: 123})
-      #=> :ok
+      iex> Honeybadger.notify(%RuntimeError{}, %{culprit_id: 123})
+      :ok
   """
-  @spec notify(Exception.t, Map.t, list()) :: :ok
-  def notify(exception, metadata \\ %{}, stacktrace \\ nil) do
+  @spec notify(Notice.noticeable, Map.t, list) :: :ok
+  def notify(exception, metadata \\ %{}, stacktrace \\ nil)
+      when is_map(exception) or is_binary(exception) do
```

Comment:
I was a little confused by the line break/indentation of the guard clause here. Is this format standard?

Reply:
That is the standard format and the way the new built in formatter will indent guard clauses that exceed the maximum line width.